client/src/components/Page/index.tsx

```diff
@@ -21,7 +22,18 @@ interface Props extends WithStyles<typeof styles> {
 }
 
 function Page({ classes, children }: Props) {
-  return <div className={classes.page}>{children}</div>;
+  const [isAuthPanelOpen, setIsAuthPanelOpen] = useState(false);
+  const openAuthPanel = () => {
+    setIsAuthPanelOpen(true);
+    console.log({ isAuthPanelOpen });
```

Comment:
This is just a stub. I'm thinking the AuthPanel would be mounted on the Page component so it can be opened from any page?

Reply:
sgtm!

Reply:
Makes sense 2 me